engine/access/rpc/backend/backend_test.go

```diff
@@ -602,7 +602,7 @@ func (suite *Suite) TestGetEventsForBlockIDs() {
 		// create receipt mocks that always returns empty
 		receipts := new(storagemock.ExecutionReceipts)
 		receipts.
-			On("ByBlockIDAllExecutionReceipts", mock.Anything).
+			On("ByBlockID", mock.Anything).
 			Return([]*flow.ExecutionReceipt{}, nil).Once()
```

Comment:
The mock implements differently than the actual implementation which will return `nil, storage.ErrNotFound`